apps/slackbot/user/user_usage_tracker.py

```diff
@@ -0,0 +1,175 @@
+import sqlite3
+import time
+
+
+class UserUsageTracker:
```

Comment:
Also, we might want to call the folder `database` (instead of `db`) so we can put any other database related things here